fully_documented.txt

```diff
@@ -4,6 +4,9 @@ backpack/core/derivatives/basederivatives.py
 backpack/core/derivatives/rnn.py
 backpack/core/derivatives/shape_check.py
 backpack/core/derivatives/__init__.py
+backpack/core/derivatives/permute.py
+
+backpack/extensions/backprop_extension.py
```

Comment:
Move to line 25 to keep directories grouped

Reply:
Can you please double-check whether it's really better?
I feel like grouping all of `backpack/extensions/` together would go too far. I prefer having the sub-folders of it separate.